project/Dependencies.scala

```diff
@@ -6,9 +6,10 @@ object Dependencies {
   lazy val lambdaJavaCore = "com.amazonaws" % "aws-lambda-java-core" % "1.2.1"
   lazy val lambdaJavaEvents = "com.amazonaws" % "aws-lambda-java-events" % "3.1.0"
   lazy val s3Sdk = "software.amazon.awssdk" % "s3" % "2.13.18"
-  lazy val ses = "software.amazon.awssdk" % "ses" % "2.15.2"
+  lazy val sesSdk = "software.amazon.awssdk" % "ses" % "2.15.2"
   lazy val sqsSdk = "software.amazon.awssdk" % "sqs" % "2.13.15"
   lazy val ecrSdk = "software.amazon.awssdk" % "ecr" % "2.13.15"
+  lazy val sfnSdk = "software.amazon.awssdk" % "sfn" % "2.15.79"
```

Comment:
Is there a particular reason for using different versions of all the AWS SDKs? Or could we have a variable for the version number so we can upgrade them all at once?

Reply:
Nope, I just grabbed the latest version of the library from the maven repository. I'll put in a variable.